dll/win32/shell32/utils.cpp

```diff
@@ -9,6 +9,46 @@
 
 WINE_DEFAULT_DEBUG_CHANNEL(shell);
 
+/*************************************************************************
+ *                SHGetShellStyleHInstance (SHELL32.749)
+ */
+EXTERN_C HINSTANCE WINAPI SHGetShellStyleHInstance(VOID)
+{
+    HINSTANCE hInst = NULL;
+    WCHAR szPath[MAX_PATH], szColorName[100];
+    HRESULT hr;
+    CStringW strShellStyle;
+
+    TRACE("\n");
+
+    /* First, attempt to load the shellstyle dll from the current active theme */
+    hr = GetCurrentThemeName(szPath, _countof(szPath), szColorName, _countof(szColorName), NULL, 0);
+    if (FAILED(hr))
+        goto DoDefault;
+
+    /* Strip the theme filename */
+    PathRemoveFileSpecW(szPath);
+
+    strShellStyle = szPath;
+    strShellStyle += L"\\Shell\\";
+    strShellStyle += szColorName;
+    strShellStyle += L"\\ShellStyle.dll";
```

Comment:
I wonder how all this behaves in a case-sensitive environment like ROS installed in BTRFS (not that we support shellstyle dll files in any way, but still...)